src/components/SiteSwitcher.tsx

```diff
@@ -26,16 +26,22 @@ export const SiteSwitcher = () => {
 
   return (
     <Flex align="center" justify="space-between" w="100%" ff="Lato">
-      <Text fz="14px" pr="md" c="#BFC1C1" className="hidden sm:block">
+      <Text
+        fz="14px"
+        pr="md"
+        c="#BFC1C1"
+        display={{ base: "none", sm: "block" }}
+      >
         Switch to different test shops
       </Text>
 
-      <ButtonGroup variant="outline">
+      <ButtonGroup variant="outline" w={{ base: "100%", sm: "auto" }}>
         {SITES.map((site) => {
           const active = currentSite === site.key
 
           return (
             <Button
+              w={{ base: "100%", sm: "auto" }}
```

Comment:
It seems the combination of `w` on the `ButtonGroup` and here accidentally got rid of the left button rounded border.

<img width="709" alt="Screenshot 2024-10-07 at 2 08 47 PM" src="https://github.com/user-attachments/assets/0c2c9422-deb3-4981-9b49-7457841cfe39">

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/b10750cc499f1836f48a2187c687a3b52743a814 by using CSS, to avoid Mantine prepending the `<style>` tag to the first button